### PR TITLE
Feat/date managment page

### PR DESCRIPTION
**Pull request by Parle648**
## Describe your changes

- Create first stage page (UI + Connect Fe to Be)

## Issue ticket code and link

- [Link to JIRA ticket](https://codecraftersintership.atlassian.net/browse/SCRUM-42)

image

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [x] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [x] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/components/ExampleComponent/styles.ts
Comment (lyudad) [inline src/components/ExampleComponent/styles.ts:1]: удаляй этот компонент полностью

File: src/pages/ExamplePage/index.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 
 import ExampleComponent from '@/components/ExampleComponent';
-import useExamplePageStyles from '@/pages/ExamplePage/styles';
+// import useExamplePageStyles from '@/pages/ExamplePage/styles';
```
Comment (lyudad) [inline src/pages/ExamplePage/index.tsx]: 1.1.10

File: src/components/ExampleComponent/styles.ts
Comment (Parle648) [reply src/components/ExampleComponent/styles.ts:1]: done

File: src/pages/ExamplePage/index.tsx (hunk 1 above)
Comment (Parle648) [reply src/pages/ExamplePage/index.tsx]: deleted

**Pull request merged**